app/views/cashflows/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="w-full">
+  <% if notice.present? %>
+    <p class="py-2 px-3 bg-green-50 mb-5 text-green-500 font-medium rounded-lg inline-block" id="notice"><%= notice %></p>
+  <% end %>
+
+  <div class="flex justify-between items-center">
+    <h1 class="font-bold text-4xl">All transactions</h1>
+  </div>
+
+  <div id="cashflows" class="min-w-full">
+    <div class="container mt-10">
+    <% if params[:id].present?%>
+      <%= link_to "Add Transaction", new_category_cashflows_path(params[:id]), class: "ml-2 rounded-lg py-1 px-2 bg-gray-100 font-medium flex-1" %>
+    <%end%>  
+    <ul>
+      <% @cashflows.each do |cashflow|%>
+        <li>
+          <h2><%= cashflow.name %></h2>
+          <p>USD <%= cashflow.amount %></p>
+          <small>Added at: <%= cashflow.created_at%></small>
+        </li>
+      <% end %>
```

Comment:
- [ ] Kindly re-design this page to look exactly like the given design. You should put the back button on the top so that users can use it to go back to the categories page.


![trasactions](https://github.com/Reggeditt/tranzact/assets/53124548/bcc2f53a-8c66-4dc4-8c7e-acb99cf367bf)


![trpage](https://github.com/Reggeditt/tranzact/assets/53124548/c5d282da-a687-4987-a593-7025180d8ff7)

Reply:
Can be improved!